circle.yml

```diff
@@ -1,6 +1,7 @@
 machine:
   environment:
     TWILIO_PHONE_NUMBER: +12435551212
+    TWILIO_AUTH_TOKEN: 37a9858200d9f5dfd6c47d2630a40f8697
```

Comment:
Hah, one downside of the randomly generated keys here is that I never know if it is the actual key or not 📦